cdap-docs/developers-manual/source/advanced/etl.rst

```diff
@@ -0,0 +1,85 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _advanced-custom-etl:
+
+===============================================
+Creating Custom ETL Applications and Components
```

Comment:
What are ETL Applications? Are you referring to creating ETL Plugins? 

It should be called 
"Creating Custom ETL Plugins"